### PR TITLE
Initialise _showWidget in FavoritesManager

### PR DESCRIPTION
**Pull request by mctoohey**
`this._showWidget` can be undefined when the plugin is initially loading. Which causes `isVisible` to return undefined. Which in turn  can cause the widget to never render because React 17 and below will error if `undefined` is returned in the render function. Specifically, this part of the will cause an error if `isVisible` is undefined.
```
{(manager: FavoritesManager, isVisible: boolean) =>
                isVisible && (
                  <>
                    <div className={FAVORITE_HEADER_CLASS}>Favorites</div>
                    <div className={FAVORITE_CONTAINER_CLASS}>
                      {visibleFavorites.map((f) => (
                        <FavoriteComponent
                          key={`favorites-item-${f.path}`}
                          favorite={f}
                          handleClick={manager.handleClick.bind(manager)}
                        />
                      ))}
                    </div>
                    <div className={FILEBROWSER_HEADER_CLASS}>File Browser</div>
                  </>
                )
              }
 ```

I think this has always been a race condition but was very rare. However, since Jupyterlab `v4.0.0a27` I have found that this issue happens consistently and the Favorites never renders widget.

### CODE REVIEW

**Comment by mctoohey** [comment]
@fcollonval When you have some spare time, would you be able to give this a review?

File: src/manager.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ export class FavoritesManager {
   }
 
   isVisible() {
-    return this._showWidget && this.visibleFavorites().length > 0;
+    return Boolean(this._showWidget && this.visibleFavorites().length > 0);
```
Comment (fcollonval) [inline src/manager.ts:81]: Thanks @mctoohey would you mind changing the fix to set `this._showWidget = true;` in the constructor of `FavoritesManager`? That will be more robust.

**Review by fcollonval — CHANGES_REQUESTED** [review]

**Comment by mctoohey** [comment]
Thanks, I've updated the original commit to initialise `this._showWidget = true;`

**Review by fcollonval — APPROVED** [review]
Thanks @mctoohey

**Pull request merged**